Source/WebKit/UIProcess/API/Cocoa/WKWebViewConfiguration.mm

```diff
@@ -458,6 +466,9 @@ - (id)copyWithZone:(NSZone *)zone
 
     configuration->_allowsInlinePredictions = self->_allowsInlinePredictions;
 
+    configuration->_scrollToTextFragmentMarkingEnabled = self->_scrollToTextFragmentMarkingEnabled;
+    configuration->_scrollToTextFragmentMarkingEnabled = self->_scrollToTextFragmentMarkingEnabled;
```

Comment:
I think you meant to use `_scrollToTextFragmentIndicatorEnabled` on one of these lines 😅